cinema/serializers.py

```diff
@@ -22,3 +27,48 @@
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for the `ActorSerializer` class to simplify the code by automatically handling the `create` and `update` methods.